Std/Data/BitVec/Basic.lean

```diff
@@ -433,3 +433,21 @@ def signExtend (v : Nat) (x : BitVec w) : BitVec v := .ofInt v x.toInt
 @[simp] theorem add_eq (x y : BitVec w)                   : BitVec.add x y = x + y            := rfl
 @[simp] theorem sub_eq (x y : BitVec w)                   : BitVec.sub x y = x - y            := rfl
 @[simp] theorem mul_eq (x y : BitVec w)                   : BitVec.mul x y = x * y            := rfl
+
+/-- Turn a `Bool` into a bitvector of length `1` -/
+def ofBool : Bool → BitVec 1
+  | false => 0
+  | true  => 1
+
+/-- The empty bitvector -/
+def nil : BitVec 0 :=
+  BitVec.zero 0
```

Comment:
```suggestion
abbrev nil : BitVec 0 :=0
```

Reply:
It looks like we're missing `zero_eq` in the lemmas above.

Reply:
What do you mean by `zero_eq`, what would be its lhs and rhs?

Reply:
` theorem zero_eq : BitVec.zero n = 0 := rfl`

Reply:
Added it (with `0#n` as the rhs, since we also have a `0 = 0#n` simp lemma already)